tables/rx_potency/chen21e-pot.csv

```diff
@@ -0,0 +1,118 @@
+ref_name,rx_name,iso_name,section,assay_name,potency_type,potency,cumulative_count,potency_upper_limit,potency_lower_limit,potency_unit,date_added
+Chen21e,COV2-2130,D614G,Figure 1c,Virus isolate,IC50,26,1,10000,1,ng/ml,1/8/2022
```

Comment:
Since I am not able to comment in the `isolates.csv` file due to the CSV format issue, I'll leave the comments here.

We have a naming rule for isolates. `D614G` should be `S:614G`.